apps/src/code-studio/components/progress/UnitCalendarLessonChunk.jsx

```diff
@@ -0,0 +1,150 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+const styles = {
+  common: {
+    margin: 5,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    textAlign: 'center',
+    color: '#333',
+    textDecorationLine: 'none',
+    fontFamily: '"Gotham 4r", sans-serif'
+  },
+  assessment: {
+    border: '2px solid rgb(118, 101, 160)'
+  },
+  assessmentHover: {
+    border: '2px solid rgb(118, 101, 160)',
+    backgroundColor: 'rgb(118, 101, 160)',
+    color: 'white'
+  },
+  instructional: {
+    border: '2px solid #00adbc'
+  },
+  instructionalHover: {
+    border: '2px solid #00adbc',
+    backgroundColor: '#00adbc',
+    color: 'white'
+  },
+  isNotStart: {
+    borderLeftStyle: 'dashed'
+  },
+  isNotEnd: {
+    borderRightStyle: 'dashed'
+  },
+  titleText: {
+    width: '100%',
+    paddingRight: 15
+  },
+  iconSection: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    height: '100%',
+    paddingLeft: 2,
+    paddingTop: 2,
+    paddingBottom: 2,
+    boxSizing: 'border-box'
+  }
+};
+
+export default class UnitCalendarLessonChunk extends Component {
+  static propTypes = {
+    minuteWidth: PropTypes.number.isRequired,
+    lesson: PropTypes.shape({
```

Comment:
I think you used this shape in multiple places. It would be good to make a file to define the Unit Calendar shapes if they are reused. Check out the https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/lessonOverview/lessonPlanShapes.jsx as an example